playbooks/appsemblerPlaybooks/update_custom_requirements.yml

```diff
@@ -0,0 +1,106 @@
+- hosts: "edxapp-server"
+  sudo: true
```

Comment:
`sudo` is deprecated. Should use `become: true`. See here: http://docs.ansible.com/ansible/latest/become.html#become

Reply:
Ahh, right we're still on 1.9 because of Eucalyptus. Sudo still works, so matbe just leave it and do the change when this is ported to Ficus+